addon/components/ember-thead/component.js

```diff
@@ -214,9 +221,10 @@ export default Component.extend({
     this._updateApi();
     this._updateColumnTree();
 
+    addObserver(this, 'enableScrollIndicators', this._updateApi);
+    addObserver(this, 'reorderFunction', this._updateApi);
```

Comment:
Is there anything significant about the reorder of observer initialization here?

Reply:
I was just trying to alphabetize this group of observers rather than continue appending new ones after old ones.